### PR TITLE
Set up CI with Azure Pipelines

### PR DESCRIPTION
**Pull request by millicentachieng**

### CODE REVIEW

File: Test-Docs.ps1 [hunk 1]
```diff
@@ -2,67 +2,93 @@ Param(
...
 
 Write-Host "Repository location: ", $repoPath
 
+# Check if ApiDoctor version has been set
+if ([string]::IsNullOrWhiteSpace($apiDoctorVersion)) {
+	Write-Host "API Doctor version has not been set. Aborting..."
+	exit 1
+}
+
+# Get NuGet
```
Comment (daspek) [inline Test-Docs.ps1]: move this into the section before it uses nuget. right now it's going to download nuget all the time, even if you're going to end up using git

File: Test-Docs.ps1 [hunk 2]
```diff
@@ -2,67 +2,93 @@ Param(
     [switch]$cleanUp,
     [string]$file
 )
+$apiDoctorVersion = $env:APIDOCTOR_VERSION
```
Comment (daspek) [inline Test-Docs.ps1]: does this need to go into the `azure-pipelines.yml` file?

File: Test-Docs.ps1 (hunk 1 above)
Comment (millicentachieng) [reply Test-Docs.ps1]: @daspek I am using nuget to restore packages before building solution with MSBuild on line 51

File: Test-Docs.ps1 (hunk 2 above)
Comment (millicentachieng) [reply Test-Docs.ps1]: Not necessarily. I thought we could set it as pipeline variable on the designer interface. That way we do not need to make changes to .yml file each time we want to use a different version of api doctor unless it's important to track that on source control.

**Pull request merged**